app/assets/stylesheets/styles.scss

```diff
@@ -1,3 +1,8 @@
+$junk_color: #6e6e6e;
+$heirloom_color: #04b404;
```

Comment:
Name of variable `heirloom_color` should be written in all lowercase letters with hyphens instead of underscores